modify partials generation scripts (#252)
loft-sh/vcluster-docs: platform/api/_partials/resources/config/external/platform/autoSleep/autoWakeup.mdx

<details className="config-field" data-expandable="true">
<summary>

## `autoWakeup` <span className="config-field-required" data-required="false">required</span> <span className="config-field-type">object</span> <span className="config-field-default"></span> <span className="config-field-enum"></span> <span data-pro="false" className="config-field-pro">pro</span> {#autoWakeup}

AutoSleep holds configuration for automatic wakeup

</summary>

<details className="config-field" data-expandable="false" open>
<summary>

### `schedule` <span className="config-field-required" data-required="false">required</span> <span className="config-field-type">string</span> <span className="config-field-default"></span> <span className="config-field-enum"></span> <span data-pro="false" className="config-field-pro">pro</span> {#autoWakeup-schedule}

Schedule specifies scheduled wakeup from sleep in Cron format, see https://en.wikipedia.org/wiki/Cron.

> Note: timezone defined in the schedule string will be ignored. The timezone for the autoSleep schedule will be
> used

Reviewer: ```suggestion
:::note
If a timezone is set in the autoSleep schedule, any timezone defined in the schedule string isignored.
:::
```

Author: changed :+1:
